### PR TITLE
Remove & Rename Errors and Correct Names

### PR DESCRIPTION
**Pull request by jdubpark**
This PR
- Removes unused Errors from beta and v1.0.0-rc.1
- Rename some Errors for unified naming
- Corrects some variable names
- Removes some unnecessary comments

### CODE REVIEW

**Review by LeoHChen — APPROVED** [review]

File: contracts/lib/Errors.sol [hunk 1]
```diff
@@ -71,297 +57,454 @@ library Errors {
...
+    /// @notice Provided parent IP and the parent dispute's target IP is different.
     error DisputeModule__ParentIpIdMismatch();
+
+    /// @notice Provided parent dispute's target IP is not dispute tagged.
     error DisputeModule__ParentNotTagged();
+
+    /// @notice Provided parent dispute's target IP is not the derivative IP's parent.
     error DisputeModule__NotDerivative();
+
+    /// @notice Provided parent dispute is still in dispute state.
```
Comment (Spablob) [inline contracts/lib/Errors.sol:271]: Provided parent dispute has not been resolved.

File: contracts/lib/Errors.sol [hunk 2]
```diff
@@ -71,297 +57,454 @@ library Errors {
...
-    error RoyaltyPolicyLAP__ZeroLiquidSplitMain();
-    error RoyaltyPolicyLAP__NotRoyaltyModule();
+
+    /// @notice Zero address provided for Licensing Module.
     error RoyaltyPolicyLAP__ZeroLicensingModule();
+
+    /// @notice Caller is not the Royalty Module.
+    error RoyaltyPolicyLAP__NotRoyaltyModule();
+
+    /// @notice Size of parent IP list is above the protocol limit.
```
Comment (Spablob) [inline contracts/lib/Errors.sol:358]: Size of parent IP list is above the LAP royalty policy limit.

File: contracts/lib/Errors.sol [hunk 3]
```diff
@@ -71,297 +57,454 @@ library Errors {
...
     error RoyaltyPolicyLAP__AboveParentLimit();
+
+    /// @notice Amount of ancestors for derivative IP is above the protocol limit.
```
Comment (Spablob) [inline contracts/lib/Errors.sol:361]: Amount of ancestors for derivative IP is above the LAP royalty policy limit.

File: contracts/utils/ShortStringOps.sol [hunk 4]
```diff
@@ -9,45 +9,8 @@ library ShortStringOps {
     using ShortStrings for *;
```
Comment (Spablob) [inline contracts/utils/ShortStringOps.sol:9]: optional: we can move ShortStringOps.sol library to the "lib" folder and delete the current folder where it is in

**Review by Spablob — APPROVED** [review]

File: contracts/lib/Errors.sol (hunk 3 above)
Comment (jdubpark) [reply contracts/lib/Errors.sol]: [9736b8e](https://github.com/storyprotocol/protocol-core-v1/pull/112/commits/9736b8e55e96c43721643e61a88f491c51b900b9)

File: contracts/lib/Errors.sol (hunk 1 above)
Comment (jdubpark) [reply contracts/lib/Errors.sol]: [9736b8e](https://github.com/storyprotocol/protocol-core-v1/pull/112/commits/9736b8e55e96c43721643e61a88f491c51b900b9)

File: contracts/utils/ShortStringOps.sol (hunk 4 above)
Comment (jdubpark) [reply contracts/utils/ShortStringOps.sol]: 1aa42a9

**Pull request merged**